### PR TITLE
Created an AllCaps format test (Issue #237)

### PR DESCRIPTION
**Pull request by MoromugHSE**
Created (and updated to use current number of available metrics) a test suggested by issue #237.

### CODE REVIEW

File: tests/steps/test-allcaps.sh [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        Foo(String zz) {
+            this.z = zz;
+        }
+        private final boolean boom() { return true; }
+    }
+EOT
+    msg=$("${LOCAL}/steps/measure-file.sh" "${java}" "${temp}/m1")
+    set -x
+    all=$(find "${temp}" -name 'm1.*' -type f -exec basename {} \; | sort)
+    expected=52
```
Comment (yegor256) [inline tests/steps/test-allcaps.sh:46]: @MoromugHSE looks like a typo. The `expected` should be counted from `all`

**Comment by yegor256** [comment]
@MoromugHSE the test looks good, but it will fail, I believe. We should fix the code, in order for this test to work.

**Comment by MoromugHSE** [comment]
Well, nothing fails on my local machine now! Seems to be all great.

**Comment by MoromugHSE** [comment]
Now that is kind of strange. I don't see anything I could've done to destroy those package upgrades...

**Comment by yegor256** [comment]
@MoromugHSE there is an obvious error in the build:

```
Script '/cam/tests/steps/test-allcaps.sh' is not executable, try running 'chmod +x /cam/tests/steps/test-allcaps.sh'
```

**Comment by yegor256** [comment]
@MoromugHSE thanks!

**Pull request merged**